xc/xc7/toolchain_wrappers/write_bitstream

```diff
@@ -7,7 +7,16 @@ MYPATH=`dirname ${MYPATH}`
 source ${MYPATH}/env
 echo "Writing bitstream ..."
 
-FASM2FRAMES=`realpath ${VPRPATH}/python/prjxray/utils/fasm2frames.py`
+if [ -z ${XRAY_FASM2FRAMES} ]; then
+	echo "ERROR: XRAY_FASM2FRAMES variable is unset."
+	return 1
+fi
+
+if [ -z ${XRAY_DATABASE_DIR} ]; then
+	echo "ERROR: XRAY_DATABASE_DIR variable is unset."
+	return 1
+fi
+
 FRAMES2BIT=`realpath ${VPRPATH}/xc7frames2bit`
```

Comment:
This is wrong now?  If the plan is to package prjxray in a seperate conda package, this should point to that, not to the binary from symbiflow-arch-defs?